utils/attributes.go

```diff
@@ -44,7 +44,6 @@ func StringsMatchingWildcards(attributes, wildcards []string) ([]string, error)
 						wildcard)
 				}
 				b.WriteString(wildcard[i : i+2])
-				i++
```

Comment:
I don't think this is correct, because this case consumes two characters (btw, Github's buggy tab rendering is extremely confusing here). But there may indeed be a bug here. I think the correct change is:

```diff
-		for i := range wildcard {
+		for i := 0; i < len(wildcard); i++ {
```

Reply:
I think you are right! I reverted and modified the for loop instead. I'll rerun all tests just to be sure...